3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -26,12 +26,12 @@ const rollDice = require('../../helpers/pokerDiceRoller');
 function rollTheDices() {
   // TODO Refactor this function
   const dices = [1, 2, 3, 4, 5];
-  return rollDice(1);
+  return Promise.all( dices.map(dice => rollDice(dice)) );
```

Comment:
Can be shortened to:

```suggestion
  return Promise.all(dices.map(rollDice);
```

In general, any function that has the pattern below when passed as a parameter to another function

```js
(args) => someFunction(args)
```

can be replaced with

```js
someFunction
```

This is because both forms have the same call signature.